index.d.ts

```diff
@@ -111,6 +111,12 @@ export interface FlatGridProps<ItemType = any> {
    * falls back to using the index, like React does.
    */
   keyExtractor?: (item: ItemT[], index: number) => string;
+
+  /**
+   * This prop can be used when the same Grid, component is used several times inside the cell of
```

Comment:
Please improve this description. Something like
`A unique identifier for the Grid. This key is necessary if you are nesting multiple FlatGrid/SectionGrid inside another Grid (or any VirtualizedList)`